data/Place.csv

```diff
@@ -81,7 +81,7 @@ PL0079,Saratow,51.592365,45.960804,2017-09-29,MG,2017-09-29,MG,
 PL0080,Shaoshan,27.905996376,112.474664768,2017-10-03,MG,2017-10-03,MG,
 PL0081,Huangbei,29.758889,118.534167,2017-10-03,MG,2017-10-03,MG,
 PL0082,Suzhou,31.30408,120.59538,2017-10-03,MG,2017-10-03,MG,
-PL0083,Landport,50.803000,-1.087000,2017-10-08,MG,2017-10-08,MG,
+PL0083,Landport,50.803,-1.087200,2017-10-08,MG,2017-10-08,MG,
```

Comment:
there is still auto-formatting going on here, but db can handle this (as long as the `.` don't disappear again.